fn foo() {
    x = 10; //~ ERROR cannot
}
